### DISCUSSION TITLE
How to download CutePet?

### DISCUSSION THREAD

**Discussion by benny98tr** [BennyGaming635/My-C--Projects#4] [Q&A] [answered]
How do I download CutePet? I wanna play it but I can't seem to find the installer. When I download it the download is blocked and I get a screen saying 'Download blocked'. It also says 'Ask your admin and reference this code (REDACTED)'.

Thanks!

**Comment by BennyGaming635** [comment, marked as answer]
Hi @benny98tr!

It seems that you're at either a school or workspace. As your error description says, please ask your admin and reference your redacted error code. Most likely they blocked GitHub downloads or .exe downloads. :)
